src/components/CurrentWeather.vue

```diff
@@ -30,35 +42,79 @@
 <script lang="ts">
 import { computed, defineComponent, ref } from 'vue';
 import { useWeatherStore } from '../stores/weather';
+import { useCitiesStore } from "../stores/cities";
 
 export default defineComponent({
 	setup() {
-		const store = useWeatherStore();
+		const weatherStore = useWeatherStore();
+		const citiesStore = useCitiesStore();
 		const city = ref('');
 		const hasSearched = ref(false);
+		const showSuggestions = ref(true);
 
-		const getWeather = async (): Promise<void> => {
+		const getWeather = async () => {
 			hasSearched.value = true;
-			store.setLoading(true);
+			weatherStore.setLoading(true);
 			try {
-				const response: any = await store.fetchWeather(city.value);
-				store.setWeather(response.data);
+				const response: any = await weatherStore.fetchWeather(city.value);
+				weatherStore.setWeather(response.data);
 			} catch (error: any) {
-				store.setError(error.message);
+				city.value = '';
 			} finally {
-				store.setLoading(false);
+				weatherStore.setLoading(false);
 			}
 		};
 
+		const fetchCities = async () => {
```

Comment:
Quite long function as for component body. How about creating composable even it will be single use it potentially could be useful somewhere else in future but it also might be a good eyecatcher that you are familiar with composables :)